keyboards/snes_macropad/snes_macropad.c

```diff
@@ -0,0 +1,130 @@
+// Copyright 2023 John Barbero Unenge (@jbarberu)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+// oled keylog rendering has been kindly borrowed from crkbd <3
+
+#include QMK_KEYBOARD_H
+
+oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+    return rotation;
+}
+
+__attribute__((weak)) const char * get_layer_name_user(int layer) {
+    return "Unknown";
+}
+
+static void oled_render_layer(void) {
+    oled_write("Layer: ", false);
+    oled_write_ln(get_layer_name_user(get_highest_layer(layer_state)), false);
+}
+
+static void oled_render_keylog(void);
+
```

Comment:
This should be placed in a `snes_macropad.h` header file, or the functions in this file should be rearranged accordingly.
```suggestion
```